internal/services/oracle/oracle_database_system_data_source.go

```diff
@@ -0,0 +1,328 @@
+
+	// DatabaseSystemProperties
+	ClusterName                  string                       `tfschema:"cluster_name"`
+	ComputeCount                 int64                        `tfschema:"compute_count"`
+	ComputeModel                 string                       `tfschema:"compute_model"`
+	DatabaseEdition              string                       `tfschema:"database_edition"`
+	DataStorageSizeInGbs         int64                        `tfschema:"data_storage_size_in_gbs"`
+	DatabaseSystemOptions        []DatabaseSystemOptionsModel `tfschema:"database_system_options"`
+	DatabaseVersion              string                       `tfschema:"database_version"`
+	DiskRedundancy               string                       `tfschema:"disk_redundancy"`
+	DisplayName                  string                       `tfschema:"display_name"`
+	Domain                       string                       `tfschema:"domain"`
+	GridImageOcid                string                       `tfschema:"grid_image_ocid"`
+	Hostname                     string                       `tfschema:"hostname"`
+	LicenseModel                 string                       `tfschema:"license_model"`
+	LifecycleDetails             string                       `tfschema:"lifecycle_details"`
+	LifecycleState               string                       `tfschema:"lifecycle_state"`
+	ListenerPort                 int64                        `tfschema:"listener_port"`
+	MemorySizeInGbs              int64                        `tfschema:"memory_size_in_gbs"`
+	NetworkAnchorId              string                       `tfschema:"network_anchor_id"`
+	NodeCount                    int64                        `tfschema:"node_count"`
+	OciUrl                       string                       `tfschema:"oci_url"`
+	Ocid                         string                       `tfschema:"ocid"`
+	ResourceAnchorId             string                       `tfschema:"resource_anchor_id"`
+	ScanDnsName                  string                       `tfschema:"scan_dns_name"`
+	ScanIPs                      []string                     `tfschema:"scan_ips"`
+	Shape                        string                       `tfschema:"shape"`
+	Source                       string                       `tfschema:"source"`
+	SshPublicKeys                []string                     `tfschema:"ssh_public_keys"`
+	StorageVolumePerformanceMode string                       `tfschema:"storage_volume_performance_mode"`
+	TimeZone                     string                       `tfschema:"time_zone"`
+	Version                      string                       `tfschema:"version"`
+}
+
+type DatabaseSystemOptionsModel struct {
+	StorageManagement string `tfschema:"storage_management"`
+}
+
+func (d DatabaseSystemDataSource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"resource_group_name": commonschema.ResourceGroupNameForDataSource(),
```

Comment:
please follow the [guidance](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-new-data-source.md#step-3-scaffold-an-emptynew-data-source) to order the properties in schema and documents

<img width="766" height="160" alt="image" src="https://github.com/user-attachments/assets/517edd87-bc53-4f44-8794-40670ec71263" />